code/drasil-printers/lib/Language/Drasil/Markdown/Helpers.hs

```diff
@@ -62,7 +62,7 @@ reflinkURI ref txt = if ref == txt then ang ref
 
 -- | Helper for setting up figures
 image :: Doc -> Doc -> Doc
-image f c =  text "!" <> reflinkURI fp c $^$ bold c
+image f c =  text "!" <> reflinkURI fp (text "Figure: " <> c) $^$ bold (text "Figure: " <> c)
```

Comment:
This might just be me being picky, but the parameter for `reflinkURI` is the alt text, which I don't think should have "Figure:" appended to it. If we do, then we should add it to the HTML alt text as well.

Reply:
Sure, I will wait for the above questions to be answered and then modify it together.